src/entity/FourMultipleChoiceRound.java

```diff
@@ -0,0 +1,49 @@
+package entity;
+
+public class FourMultipleChoiceRound implements Round{
```

Comment:
This is more future food for thought, but I have a feeling that the implementations for `FourMultipleChoiceRound` and `TwoMultipleChoiceRound` are going to be almost identical since the only fundamental difference is that one has four options while the other two options. We might want to consider creating one flexible `MultipleChoiceRound` class that accepts the number of options given as a constructor parameter (or something similar). Then, creating an easy round would be as simple as creating a multiple choice round passing in 2 as the number of options while creating a medium round would be the same but passing in 4 instead. This would make creating alternate multiple choice round types more flexible as well in case we want to add more in the future.

We can worry about this more later, I just wanted to make a note of this :)

Reply:
No for sure this is a great idea. I created this file just to get the use case working and so like you said this will be a good idea when we properly implement this.